src/server/DeviceServer_v2.js

```diff
@@ -299,21 +299,21 @@ class DeviceServer {
     }
 
     // TODO figure this out
-    if (lowerEventName.indexOf('spark/device/safemode') === 0) {
-      const token = device.sendMessage('Describe');
-      const systemMessage = await device.listenFor(
-        'DescribeReturn',
-        null,
-        token,
-      );
-
-      if (global.api) {
-        global.api.safeMode(
-          deviceID,
-          systemMessage.getPayload().toString(),
-        );
-      }
-    }
+    // if (lowerEventName.indexOf('spark/device/safemode') === 0) {
```

Comment:
that code and the other one regarding the spark/device/claim/code event were introduced by me in the old client/SparkCore.js. global.api.safemode and global.api.linkDevice refer to 2 methods in views/api_v1.js in spark-server i implemented in order to manage the safemode status of a device and the claim process. also i publish the spark/device/system/version event from my devices every time they start in order to save the current firmware system version